symlink.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* eslint-disable import/no-nodejs-modules */
+
+const {exec} = require('child_process'); //
+const path = require('path');
+
+const parameters = process.argv.slice(2);
+
+const from = parameters.length > 1 ? parameters[0] : '../../vendor';
```

Comment:
is there a reason for making these configurale? can we pass an argument when requiring the script in another script? 🤔

Reply:
yes if somebody have a different directory structure it should not fail or atleast he should be able to change it:
works for me correctly:

```
        "preinstall": "node ../../vendor/sulu/sulu/preinstall.js ../../vendor node_modules/@sulu/vendor --force",
```